tedpermission/src/main/java/com/gun0912/tedpermission/PermissionBuilder.java

```diff
@@ -64,30 +79,63 @@ protected void checkPermissions() {
         intent.addFlags(Intent.FLAG_ACTIVITY_NEW_TASK);
         intent.addFlags(Intent.FLAG_ACTIVITY_NO_USER_ACTION);
         TedPermissionActivity.startActivity(context, intent, listener);
-        TedPermissionBase.setFirstRequest(context,permissions);
+        TedPermissionBase.setFirstRequest(context, permissionsAsStringArray());
```

Comment:
how about changing `setFirstRequest(Context, String[])` to `setFirstRequest(Context, List)`?

Reply:
Done! Also I decided to move `stringListToStringArray` to `ObjectUtils` class.